README.md

```diff
@@ -133,6 +133,10 @@ Edit the `.params` file and add the required settings.
 
 #### `params.sample:`
 
+* If the service credentials from IBM Watson Assistant is username/password based as below populate the username, password and workspace_id and replace `.params` file with the json
```

Comment:
Here's where to be nit-picky "credentials" is plural so "are" not "is". Also "based" would be a hyphen suffix thingy when used like this so username/password-based... but that is a little ugly so I suggested just say something simple like "if the service credentials from Wassistant contain username/password, then populate u,p and w_id in the .params file.